components/supabase/actions/select-row/select-row.mjs

```diff
@@ -5,7 +5,7 @@ export default {
   key: "supabase-select-row",
   name: "Select Row",
   type: "action",
-  version: "0.0.2",
+  version: "0.0.3",
   description: "Selects row(s) in a database. [See the docs here](https://supabase.com/docs/reference/javascript/select)",
   props: {
     supabase,
```

Comment:
```suggestion
  description: "Select a row from a table. [See the documentation here](https://supabase.com/docs/reference/javascript/select)",
```

Reply:
Changed "See documentation here" to "See the documentation here" to follow Google's Developer docs style guide.